controllers/registeredservice_controller.go

```diff
@@ -36,6 +36,15 @@ type RegisteredServiceReconciler struct {
 	Scheme *runtime.Scheme
 }
 
+func ServiceInCatalog(sc primazaiov1alpha1.ServiceCatalog, serviceName string) int {
+	for i := 0; i < len(sc.Spec.Services); i++ {
+		if sc.Spec.Services[i].Name == serviceName {
+			return i
+		}
+	}
+	return -1
```

Comment:
Can simplify this slightly:
```suggestion
	for i, service := range sc.Spec.Services {
		if service.Name == serviceName {
			return i
		}
	}
	return -1
```

Reply:
I like it, thanks